docs/source/webhooks.rst

```diff
@@ -82,7 +82,21 @@ When new data for a contact is being processed this might spawn additional ``con
 Payment success
 ***************
 
+..  deprecated:: 23.06
+    Use the payment_status_change events instead.
+
 This event means that a payment related to a form submission has been processed successfully.
 
 .. literalinclude:: example-events/payment-success.yaml
   :language: yaml
+
+
+Payment status change
+*********************
+
+This event means the status of a payment has changed to a final state. This usually means that payment has either succeeded or failed. After a payment failed the user might choose to retry which will yield more status changes for the same payment.
```

Comment:
I think "final" sounds more definitive that it actually is? The state could still change again later, not only from "failed" to "new" when re-trying the payment, but also from being initially accepted by the payment provider to has actually succeeded or failed at the bank. Not sure what would work better, though…

Reply:
I have made a new attempt. Is it clearer now?

Reply:
Yes, thank you!